docs/src/customization.md

```diff
@@ -4,42 +4,290 @@ description: Customize your drop-down menu and instant search results page.
+    suggestion: /* .... */
+  }
+});
+
+autocomplete(/* .... */);
+```
+
+You can read more about multi sources/categories autocomplete.js implementation in [this tutorial](https://www.algolia.com/doc/guides/search/auto-complete#multi-category).
+
+#### Look & feel
+
+We provide with some default CSS rules that are located in the `algolia/assets/css/algolia-autocomplete.css`. You can very easily add your own CSS rules to your theme's stylesheet.
+
+If for any reason you don't want the default stylesheet to be included, you can remove it like this:
+
+```php
+<?php
+
+/**
+ * Dequeue default CSS files.
+ *
+ * Hooked to the wp_print_styles action, with a late priority (100),
+ * so that it is after the stylesheets were enqueued.
+ */
+function my_theme_dequeue_styles() {
+  // Remove the algolia-autocomplete.css.
+  wp_dequeue_style( 'algolia-autocomplete' );
+}
+add_action( 'wp_print_styles', 'my_theme_dequeue_styles', 100 );
+```
+
+
+## Search Results Page
+
+### Introduction
 
 *The instant search results experience can be enabled in the **Search page** section of the plugin.*
 
 The instant search results page is powered by **instantsearch.js**, a JavaScript library that eases the process of building rich search experiences.
 
 Please checkout the [official documentation of the instantsearch.js website](https://community.algolia.com/instantsearch.js/). Also take a look at some examples of [what can be achieved with instantsearch.js](https://community.algolia.com/instantsearch.js/examples/).
 
-<div class="alert alert-info">The instantsearch feature will do its search on the **Searchable posts** index. If you don't have that index flagged for indexing you will be invited to index it with a notice on every page of the WordPress admin.</div>
+The instantsearch feature will do its search on the **Searchable posts** index. If you don't have that index flagged for indexing you will be invited to index it with a notice on every page of the WordPress admin.
+
+<div class="alert alert-info">Whatever changes you make, be sure to understand how the autocomplete.js library work by checking out their official documentations.</div>
+
+### Customization
+
+To start customizing your dropdown menu, copy/paste the `wp-content/plugins/algolia/templates` directory in your own theme, and rename it to `algolia`. If your theme is named `mytheme`, then the folder should look like: `wp-content/themes/mytheme/algolia`.
 
-## Customizations
+You can then edit the `algolia/instantsearch.php` file to customize the instant search results page (search widgets & page layout).
 
-<div class="alert alert-warning">You should never directly edit files of the plugin, because all changes would be lost at every update of the plugin.</div>
+<div class="alert alert-info">All instantsearch.js templates are using the underscore templating, learn more about it reading this [complete documentation](http://www.2ality.com/2012/06/underscore-templates.html).</div>
 
```

Comment:
If you copy/paste the template examples from instantsearch.js's website it will still work (because Hogan is still embedded). Here I was just thinking about using a single templating engine because while writing the associated documentation, it was too weird.